test/BlockStrings.hs

```diff
@@ -0,0 +1,42 @@
+
+-- | Regression tests for issue #20 https://github.com/hasura/graphql-parser-hs/issues/20
+
+module BlockStrings where
+
+import           Hedgehog
+import           Language.GraphQL.Draft.Parser
+import qualified Data.Text as T
+import           Data.Attoparsec.Text
+
+blockTest :: IO Bool
+blockTest = do
+  checkParallel $ Group "Test.parser.block-string.unit"
+    [ ("parses the specExample", blockParsesTo "\n    Hello,\n      World!\n\n    Yours,\n      GraphQL.\n  " "Hello,\n  World!\n\nYours,\n  GraphQL.")
+    , ("parses empty string", blockParsesTo "" "")
+    , ("parses newline", blockParsesTo "\n" "")
+    , ("parses newlines in empty block", blockParsesTo " \n   \n  \n            \n\n " "")
+    , ("parses very simples not-empty block", blockParsesTo "x" "x")
+    , ("parses block string containing a valid normal string inside", blockParsesTo "  \"i'm like a JSON string\"   " "\"i'm like a JSON string\"")
```

Comment:
I think `BlockStringValue(" \"i'm like a JSON string\"")` computes to `" \"i'm like a JSON string\""` (no indentation removed since it's on the first line).